connect/resolver_test.go

```diff
@@ -68,13 +68,17 @@ func TestConsulResolver_Resolve(t *testing.T) {
 		Proxy: &api.AgentServiceConnectProxyConfig{
 			DestinationServiceName: "web",
 		},
+		Tags: []string{
+			"test-tag",
+		},
```

Comment:
nit: we view tags as a semi-deprecated construct in Consul registrations - everything that can be done with tag can also be done with `Meta` and `Filter`. Long term we'd like to consolidate on just `Meta` but have never been able to prioritize the one or two places (e.g. DNS) where tags still do something that Meta doesn't currently.

So I'd accept this - it's not _wrong_ and we've not officially announced or documented that "tags" are deprecated, but it is for example the reason we don't support tag filters in connect endpoints etc.

Would you be open to writing this test using Meta instead? It's not a huge deal either way, but internally we'd tend towards supporting/encouraging use of meta of tags as we _might_ one day get around to deprecating and removing tags entirely!